src/main/kotlin/game/racing/RacingGameApplication.kt

```diff
@@ -1,12 +1,14 @@
 package game.racing
 
+import game.racing.controller.RacingGameController
 import game.racing.domain.CarFactory
-import game.racing.domain.RacingGame
 import game.racing.domain.RacingManager
 import game.racing.view.InputView
 
 fun main() {
-    val cars = CarFactory.generateCarByCarNames(InputView.inputCarNames())
-    val racingGame = RacingGame(RacingManager(cars), InputView.inputTryNumber())
-    racingGame.start()
+    val cars = CarFactory.generateCarByCarNames(InputView.inputCarNames().split(InputView.CAR_NAME_DELIMITER))
```

Comment:
입력을 어떻게 받든지 간에 자동차 이름들을 `List<String>`으로 만들어주는 역할은 View의 역할로 볼 수 있을 것 같아요 😃

```suggestion
    val cars = CarFactory.generateCarByCarNames(InputView.inputCarNames())
```